build.gradle

```diff
@@ -44,6 +44,7 @@ apply from: "$rootDir/dependencies.gradle"
 
 // change the ES version used by Spring Boot Dependency Management plugin
 ext['elasticsearch.version'] = revElasticSearch6
+//ext['elasticsearch.version'] = revElasticSearch7
```

Comment:
This is not needed here. Instead, can you add a comment in [build.gradle#L10](https://github.com/Netflix/conductor/blob/main/build.gradle#L10) with a note reminding that `revElasticSearch7` in `dependencies.gradle` needs to be updated when spring is upgraded.

Reply:
Reminding note added. Please, see [previous comment](https://github.com/Netflix/conductor/pull/2324#discussion_r680390401) about a preferable way to avoid transitive dependencies conflicts.

Reply:
Removed. Reminding note added.